### ISSUE TITLE
goxz v0.3.0 seems ignoring -C parameter

### ISSUE THREAD

**Issue by alanoliveira** [Songmu/goxz#11]
In version v0.2.0 the following command worked fine

```
goxz -C cli/
```

In the new version (v0.3.0) I got this error message
```
$ goxz -C cli/
[goxz] Initializing...
[goxz] Resources to include: []
[goxz] Building . for windows/amd64
[goxz] Building . for darwin/amd64
[goxz] Building . for linux/amd64
[goxz] [!!ERROR!!] go list failed with following output: "can't load package: package github.com/alanoliveira/mt_cmd: no Go files in /Users/a.oliveira/go/src/github.com/alanoliveira/mt_cmd\n"
```

But if I omit -C and pass the path at the end of the command it works fine
```
goxz  ./cli/
```

It seems that the -C parameter is being ignored.
Sorry if it is not an issue/bug.
And thanks for this wonderful tool ;)

**Comment by Songmu** [comment]
Thank you very much for the report. It is my oversight.

I fixed it at #12 and released as v0.3.1. Please try it. @alanoliveira

**Comment by alanoliveira** [comment]
Hello @Songmu, thank you very much for your quick response!
But now I got a different error.
Here is my output using the version 0.3.1, 0.3.0 and 0.2.0


```
$ tree $PWD
/Users/a.oliveira/go/src/github.com/alanoliveira/mt_cmd
└── cli
    └── main.go


$ goxz_v0.3.1 -C cli/
[goxz] Initializing...
[goxz] Resources to include: []
[goxz] [!!ERROR!!] mkdir goxz/.goxz-534058256/cli_windows_amd64: no such file or directory

$ goxz_v0.3.0 -C cli/
[goxz] Initializing...
[goxz] Resources to include: []
[goxz] Building . for windows/amd64
[goxz] Building . for linux/amd64
[goxz] Building . for darwin/amd64
[goxz] [!!ERROR!!] go list failed with following output: "can't load package: package github.com/alanoliveira/mt_cmd: no Go files in /Users/a.oliveira/go/src/github.com/alanoliveira/mt_cmd\n"

$ goxz_v0.2.0 -C cli/
[goxz] Initializing...
[goxz] Resources to include: []
[goxz] Package to build: [github.com/alanoliveira/mt_cmd/cli]
[goxz] Building github.com/alanoliveira/mt_cmd/cli for windows/amd64
[goxz] Building github.com/alanoliveira/mt_cmd/cli for darwin/amd64
[goxz] Building github.com/alanoliveira/mt_cmd/cli for linux/amd64
[goxz] Archiving cli_darwin_amd64.zip
[goxz] Archiving cli_linux_amd64.tar.gz
[goxz] Archiving cli_windows_amd64.zip
[goxz] Artifact archived to goxz/cli_darwin_amd64.zip
[goxz] Artifact archived to goxz/cli_windows_amd64.zip
[goxz] Artifact archived to goxz/cli_linux_amd64.tar.gz
[goxz] Success!
```

**Comment by Songmu** [comment]
Thanks for the helpful report.

It was a problem when the destination directory was not there.
I fixed to make sure to create destination directory at #13 and released it as v0.3.2.

How about it?

**Comment by alanoliveira** [comment]
Perfect!
Thank you very much for your attention and your amazing tool 🙇
